### PR TITLE
[13.0] [MIG] l10n_it_fiscalcode: Migration to 13.0

### PR DESCRIPTION
**Pull request by victoralmau**
Related to: https://github.com/OCA/l10n-italy/pull/1931 (14.0)

@Tecnativa TT27610

### CODE REVIEW

File: l10n_it_fiscalcode/README.rst
Comment (OpenCode) [inline l10n_it_fiscalcode/README.rst:23]: Version 13.0, not 14.0

File: l10n_it_fiscalcode/README.rst
Comment (OpenCode) [inline l10n_it_fiscalcode/README.rst:115]: Version 13.0, not 14.0

File: l10n_it_fiscalcode/i18n/am.po
Comment (OpenCode) [inline l10n_it_fiscalcode/i18n/am.po:9]: Version 13.0, not 11.0

File: l10n_it_fiscalcode/i18n/ar.po
Comment (OpenCode) [inline l10n_it_fiscalcode/i18n/ar.po:9]: Version 13.0, not 11.0

**Review by OpenCode — CHANGES_REQUESTED** [review]

File: l10n_it_fiscalcode/README.rst
Comment (victoralmau) [reply l10n_it_fiscalcode/README.rst:23]: Changed it now although but will regenerate it in merge process.

File: l10n_it_fiscalcode/README.rst
Comment (victoralmau) [reply l10n_it_fiscalcode/README.rst]: Changed it now although but will regenerate it in merge process.

File: l10n_it_fiscalcode/i18n/am.po
Comment (victoralmau) [reply l10n_it_fiscalcode/i18n/am.po:9]: It's not need to change it according to https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

File: l10n_it_fiscalcode/i18n/ar.po
Comment (victoralmau) [reply l10n_it_fiscalcode/i18n/ar.po:9]: It's not need to change it according to https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

Comment (pedrobaeza) [reply l10n_it_fiscalcode/i18n/ar.po:9]: Yes, POT file will be regenerated after merge, and PO files will be auto-update if needed.

**Review by OpenCode — APPROVED** [review]

**Pull request closed without merging**
